api/v1alpha1/directoryservice_types.go

```diff
@@ -47,8 +47,8 @@ type DirectoryServiceSpec struct {
 	//  +kubebuilder:default:="100Gi"
 	Storage string `json:"storage"`
 
-	// +kubebuilder:default:="standard"
-	StorageClassName string `json:"storageClassName"`
+	// +kubebuilder:validation:Optional
+	StorageClassName string `json:"storageClassName,omitempty"`
```

Comment:
you only need one of the two. `omitempty`  allows  you to keep consistent indentation in the struct 
```suggestion
	StorageClassName string `json:"storageClassName,omitempty"`
```